gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/handler/user/UserRequestHandler.java

```diff
@@ -53,7 +53,7 @@ public abstract class UserRequestHandler implements Handler<RoutingContext> {
     protected void redirectToPage(RoutingContext context, MultiMap queryParams, Throwable... exceptions) {
         try {
             if (exceptions != null && exceptions.length > 0) {
-                logger.debug("Error user actions : " + queryParams.get(ERROR_PARAM_KEY), exceptions[0]);
+                logger.debug("Error user actions : {}", queryParams.get(ERROR_PARAM_KEY), exceptions[0]);
```

Comment:
shouldn't there be 2 placeholders? "{} {}"

Reply:
this is from AbstractLogger:
```
private void handle2ArgsCall(Level level, Marker marker, String msg, Object arg1, Object arg2) {
        if (arg2 instanceof Throwable) {
            this.handleNormalizedLoggingCall(level, marker, msg, new Object[]{arg1}, (Throwable)arg2);
        } else {
            this.handleNormalizedLoggingCall(level, marker, msg, new Object[]{arg1, arg2}, (Throwable)null);
        }
}
```
If second argument is Throwable, then it is casted.